cloudfiles/secrets.py

```diff
@@ -8,21 +8,28 @@
 from .lib import mkdir, colorize
 
 HOME = os.path.expanduser('~')
-CLOUD_VOLUME_DIR = os.path.join(HOME, '.cloudvolume', 'secrets')
-CLOUD_FILES_DIR = os.path.join(HOME, '.cloudfiles', 'secrets')
+
+CV_HOME = os.path.join(HOME, '.cloudvolume')
+CLOUD_VOLUME_DIR = os.environ.get("CLOUD_VOLUME_DIR", CV_HOME)
+CLOUD_VOLUME_SECRETS_DIR = os.path.join(CLOUD_VOLUME_DIR, 'secrets')
+
+CF_HOME = os.environ.get("CLOUDFILES_CONFIG_DIR", HOME)
```

Comment:
why does `CLOUDFILES_CONFIG_DIR` mean the parent folder of `.cloudfiles`? Intuitively I would guess `CLOUD_FILES_DIR` should be the parent of `CLOUDFILES_CONFIG_DIR`, now the reverse is true. Besides, why not use `CLOUD_FILES_CONFIG_DIR` to make it consistent with other variables?

Reply:
I am not sure what the heck I wrote there, but it didn't make sense. Take a look at the new version I just pushed.